### PR TITLE
EVM Report for Documentation - DotMatrix

### PR DESCRIPTION
**Pull request by nblogist**
Removes stale links, added more information/recommendations in commit messages

### CODE REVIEW

File: docs/edgeware-runtime/evm/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # EVM
 
-Edgeware has a pallet that allows developers to write EVM smart-contracts. This means that you can use Edgeware as you would with Ethereum. Edgeware is fully compatibility with Ethereum's Web3 API and EVM. Here we'll walk through a few subtle differences with Ethereum. Namely that Edgeware has a Proof of Stake-based consensus mechanism. This shouldn't effect you if you're building a DeFi or NFT based application. See our related documentation on proof-of-stake. In the following sections we detail Edgeware&lt;&gt;EVM Compatibiltiy.
+Edgeware has a pallet that allows developers to write EVM smart-contracts. This means that you can use Edgeware as you would with Ethereum. Edgeware is fully compatibility with Ethereum's Web3 API and EVM. Here we'll walk through a few subtle differences with Ethereum. Namely that Edgeware has a Proof of Stake-based consensus mechanism. This shouldn't affect you if you're building a DeFi or NFT based application. See our related documentation on [proof-of-stake](https://docs.edgewa.re/edgeware-runtime/consensus). In the following sections we detail Edgeware&lt;&gt;EVM Compatibiltiy.
```
Comment (drewstone) [inline docs/edgeware-runtime/evm/README.md]:
```suggestion
Edgeware has a pallet that allows developers to write EVM smart-contracts. This means that you can use Edgeware as you would with Ethereum. Edgeware is fully compatible with Ethereum's Web3 API and EVM. Here, we'll walk through a few subtle differences between Edgeware and Ethereum. Namely, Edgeware has a Proof of Stake-based consensus mechanism. This shouldn't affect you if you're building a DeFi or NFT based application. See our related documentation on [proof-of-stake](https://docs.edgewa.re/edgeware-runtime/consensus). In the following sections we detail Edgeware&lt;&gt;EVM Compatibiltiy.
```

File: docs/edgeware-runtime/evm/evm-balances.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ When converting, also note the `gasPrice` is specified in the EVM call, as the c
 
 * Convert a Substrate address to a EVM address: `address[0..20]` — take the first 20 bytes.
 * Convert an EVM address into a Substrate address: add `evm:` prefix to data, and take the hash. **THIS IS NOT REVERSIBLE**.
-  * See: [https://github.com/paritytech/substrate/blob/master/frame/evm/src/lib.rs\#L169](https://github.com/paritytech/substrate/blob/master/frame/evm/src/lib.rs#L169)
+  * See: [https://gitlab.parity.io/parity/substrate/-/blob/16fdfc4a80a14a26221d17b8a1b9a95421a1576c/frame/evm/src/lib.rs#L167](https://gitlab.parity.io/parity/substrate/-/blob/16fdfc4a80a14a26221d17b8a1b9a95421a1576c/frame/evm/src/lib.rs#L167)
```
Comment (drewstone) [inline docs/edgeware-runtime/evm/evm-balances.md:55]: Please don't use gitlab, and plealse use main branches and not commits.

**Review by drewstone — CHANGES_REQUESTED** [review]
Needs some fixing. Thanks for the contribution!

File: docs/edgeware-runtime/evm/evm-balances.md (hunk 2 above)
Comment (ghost) [reply docs/edgeware-runtime/evm/evm-balances.md:55]: Hi, @drewstone. In this instance, I could only find it on Gitlab. What do you recommend?

**Review by Camnaz — APPROVED** [review]

**Pull request merged**